### DISCUSSION TITLE
关于 v3 和 v4 文档中 SwipeCell 滑动单元格 组件的样式变量问题

### DISCUSSION THREAD

**Discussion by aisen60** [youzan/vant#10427] [General]
我看了一下，v3 和 v4 代码中没有 **SwipeCell 滑动单元格** 样式变量的定义，而文档中还保留着这些样式变量，会误导开发者。

**SwipeCell 滑动单元格** 样式变量如下：

- `--van-switch-cell-padding-top` 
- `--van-switch-cell-padding-bottom`
- `--van-switch-cell-large-padding-top`
- `--van-switch-cell-large-padding-bottom`

v3 和 v4 文档中 **SwipeCell 滑动单元格** 组件的样式变量是否可以注释或者删除哈？如果可以，我提个pr哈~

**Comment by chenjiahan** [comment]
可以直接删除的，欢迎提 PR 喔

**Reply by aisen60** [reply]
提交了哈~
